src/test/ui/error-codes/E0025.stderr

```diff
@@ -2,8 +2,10 @@ error[E0025]: field `a` bound multiple times in the pattern
   --> $DIR/E0025.rs:8:21
    |
 LL |     let Foo { a: x, a: y, b: 0 } = x;
-   |               ----  ^^^^ multiple uses of `a` in pattern
-   |               |
+   |               ----  ^^^^
+   |               |     |
+   |               |     multiple uses of `a` in pattern
+   |               |     help: consider removing one usage of: ``a``
```

Comment:
Similar issue here, the result is

```rust
let Foo { a: x, `a`, b: 0 } = x;
```

We also can't know for sure which of the two usages was correct, so at least this would have to be `Applicability::MaybeIncorrect`.